b2share/modules/communities/views.py

```diff
@@ -127,7 +127,16 @@ def __init__(self, **kwargs):
 
     def get(self):
         """Retrieve a list of communities."""
-        raise NotImplementedError()
+        # TODO: change this to a search function, not just a list of communities
+        from .serializers import community_to_dict
+        start = request.args.get('start') or 0
+        stop = request.args.get('stop') or 20
+        community_list = Community.get_all(start, stop)
+        response = jsonify({
+            'communities': [community_to_dict(c) for c in community_list]
```

Comment:
ok, made an issue for this one
https://github.com/EUDAT-B2SHARE/b2share/issues/854